src/Stratis.Bitcoin/Utilities/ShellHelper.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Diagnostics;
+
+namespace Stratis.Bitcoin.Utilities
+{
+    public class ShellHelper : IShellHelper
+    {
+        public string RunCommand(string command, string args, bool wait = false, int timeout = 2000)
+        {
+            var startInfo = new ProcessStartInfo
+            {
+                FileName = command,
+                Arguments = args,
+                RedirectStandardOutput = wait,
+                RedirectStandardError = wait,
+                UseShellExecute = false,
+                CreateNoWindow = true,
+            };
+
+            if (!wait)
+            {
+                Process.Start(startInfo);
```

Comment:
Arent we probably exposing ourselves here to a security issues? We can then just execute any code here?